misc/python/materialize/mzcompose/services/materialized.py

```diff
@@ -165,9 +165,10 @@ def __init__(
 
         if force_migrations is not None and image is None:
             command += [
-                "--unsafe-mode",
                 f"--unsafe-builtin-table-fingerprint-whitespace={force_migrations}",
             ]
+            if not unsafe_mode:
+                command += ["--unsafe-mode"]
```

Comment:
Should this be `if unsafe_mode:`?

Reply:
No! The problem was that when unsafe_mode is set we set `--unsafe-mode twice`

Reply:
Ah, I see, we've set it above!